scripts/dockerfiles/Dockerfile.release

```diff
@@ -1,30 +1,31 @@
-FROM golang:1.13-stretch as builder
-WORKDIR /go/src/github.com/aws/amazon-vpc-cni-k8s
+ARG docker_arch
+ARG golang_image=golang:1.13-stretch
 
+FROM $golang_image as builder
+WORKDIR /go/src/github.com/aws/amazon-vpc-cni-k8s
 ARG arch
 ENV ARCH=$arch
-
+ENV GOARCH=$arch
 # Force the go compiler to use modules.
 ENV GO111MODULE=on
 ENV GOPROXY=direct
 
-# Add goimports
-RUN go get -u golang.org/x/tools/cmd/goimports
-
-# go.mod and go.sum go into their own layers.
-COPY go.mod .
-COPY go.sum .
-
-# This ensures `go mod download` happens only when go.mod and go.sum change.
+# Copy modules in before the rest of the source to only expire cache on module
+# changes:
+#
+COPY go.mod go.sum ./
 RUN go mod download
 
-COPY . .
-RUN make build-linux && make download-portmap
+COPY Makefile ./
+RUN make download-portmap
```

Comment:
This needs to be `RUN make portmap` if we rename the target.

Reply:
Is this the only other usage that needs to be updated if the name changes?

Reply:
Yes, as far as I'm aware at least. 😄

Reply:
Excellent, I'll make it so and we can always fix the target or the caller if we found out otherwise!